### PR TITLE
update geometry projects

### PR DESCRIPTION
**Pull request by MijhailVillarroel**
@carledriss
reviewed my practice geometry

### CODE REVIEW

File: src/main/java/com/fundacionjala/figure/Circle.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package com.fundacionjala.figure;
+
+/**
+ * Created by mijhailvillarroel on 6/16/2016.
+ */
+public class Circle extends Shape {
+
+    public static final double PI = Math.PI;
```
Comment (carledriss) [inline src/main/java/com/fundacionjala/figure/Circle.java]: This constant is unnecessary
You can use directly Math.PI

File: src/main/java/com/fundacionjala/figure/Rectangle.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        this.sideA = sideA;
+        this.sideB = sideB;
+    }
+
+    public double calculateArea() {
+        return sideA * sideB;
+    }
+
+    public double calculatePerimeter() {
+        return (FACTOR_FORMULA * sideA) + (FACTOR_FORMULA * sideB);
```
Comment (carledriss) [inline src/main/java/com/fundacionjala/figure/Rectangle.java]: Optimize the formula

``` java
return FACTOR_FORMULA * (sideA + sideB);
```

File: src/test/java/com/fundacionjala/figure/TestCircle.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by mijhailvillarroel on 6/16/2016.
+ */
+public class TestCircle {
+
+    public static final double RADIO = 5.5;
```
Comment (carledriss) [inline src/test/java/com/fundacionjala/figure/TestCircle.java]: This should be private

**Comment by MijhailVillarroel** [comment]
reviewed my practice geometry
@carledriss

**Comment by carledriss** [comment]
@MijhailVillarroel 
It is missing the javadocs. comments

after that looks good to me

**Comment by MijhailVillarroel** [comment]
@carledriss 
add javadocs comments in all class

File: src/test/java/com/fundacionjala/figure/TestSquare.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+    @Before
+    public void setUp() {
+        final double side = 5;
+        square = new Square(side);
+    }
+
+    @Test
+    public void testCalculateArea() {
+        double expectResultArea = 25;
```
Comment (carledriss) [inline src/test/java/com/fundacionjala/figure/TestSquare.java:24]: It should be final

File: src/test/java/com/fundacionjala/figure/TestSquare.java [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+        assertEquals(expectResultArea, square.calculateArea(), DELTA);
+    }
+
+    @Test
+    public void testCalculatePerimeter() {
+        double expectResultPerimeter = 20;
```
Comment (carledriss) [inline src/test/java/com/fundacionjala/figure/TestSquare.java:30]: It should be final

**Comment by carledriss** [comment]
@MijhailVillarroel 
Looks good to me!

**Pull request merged**